app-backend/src/main/kotlin/org/example/appbackend/dto/FeedbackDto.kt

```diff
@@ -0,0 +1,14 @@
+package org.example.appbackend.dto
+
+import org.example.appbackend.entity.FeedbackEntity
+
+data class FeedbackDto(
+    val name: String?,
+    val email: String?,
+    val phone: String?,
+    val companyName: String?,
+    val rating: Int?,
+    val comment: String?
+)
+
+fun FeedbackEntity.toDto() = FeedbackDto(name, email, phone, companyName, rating, comment)
```

Comment:
this staff is autogenerated using mapper (mappers packege) (slightly better design + unification with other entity-dto mapping)

Reply:
thank you!